samples/server/go-server/api/server/storage/memstore.go

```diff
@@ -42,53 +43,62 @@ func NewMemStore() server.Storager {
 		occurrencesByID: map[string]*pb.Occurrence{},
 		notesByID:       map[string]*pb.Note{},
 		opsByID:         map[string]*opspb.Operation{},
```

Comment:
What are your plans for pagination on these? We can get all of the values for a project and sort them, I guess.

Reply:
Will add pagination for them as well. Just wanted the projects to be done first. And yes, if we sort them we should be fine.